s3watcher/lib/Metrics.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    create: function (uploadResult) {
+        const metricName = uploadResult.succeeded ?
+            "UploadedImages" : "FailedUploads";
+
+        const dimensions = [{
+            "Name" : "UploadedBy",
+            "Value" : uploadResult.uploadedBy
+        }]
+
+        const timestamp = new Date;
+
+        return {
```

Comment:
If I remember somewhere - CloudWatch can be throttled.

Would it be an idea to store a queue here and push out data points on a certain size or time the way we do in the other apps?

Or would that not work as we're lambdering?